model-optimizer/extensions/front/kaldi/replace_lstm_nonlinearity.py

```diff
@@ -40,11 +40,10 @@ def run_before(self):
 
     def replace_op(self, graph: Graph, node: Node):
         # split input to (i_part, f_part, c_part, o_part, ct_1)
-        split_node_axis = Const(graph, {'value': np.int64(1)}).create_node()
-        split_node = Split(graph, {'name': 'Split_lstm_input_',
-                                   'num_splits': 5}).create_node()
+        split_node = create_op_with_const_inputs(graph, Split, {1: np.int64(1)},
+                                                 {'name': node.soft_get('name', node.id) + '/split_lstm_input',
+                                                  'num_splits': 5})
         node.in_port(0).get_connection().set_destination(split_node.in_port(0))
-        split_node.in_port(1).connect(split_node_axis.out_port(0))
 
         # i_t = Sigmoid(i_part + w_ic*ct_1)
         i_scale_attrs = {'name': 'i_scaleshift',
```

Comment:
Update name for `i_scale`, `sum_i_c`, `i_sigmoid`, `f_scale_attrs` and others below

Reply:
Done.